docs/content/docs/templates.md

```diff
@@ -897,6 +897,9 @@ Example:
 #### iterable
 Returns true if the given variable can be iterated over in Tera (ie is an array/tuple).
 
+#### object
+Returns true if the given variable is a hash (ie can be iterated over key, value).
```

Comment:
is a hash -> is an object
A hash would be a string :)

Reply:
Done